rules/php80/tests/Rector/Class_/ClassPropertyAssignToConstructorPromotionRector/parameter-is-not-passed-as-reference.php.inc

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Rector\Php80\Tests\Rector\Class_\ClassPropertyAssignToConstructorPromotionRector\Fixture;
+
+class X {
```

Comment:
@samsonasik Hi, I guess these miss-names in fixture should be reported in CI, right? 
Is it working correctly?

Reply:
it seems because it uses dash instead of underscore, so it not detected

Reply:
I see. How can we deal with those too?

Reply:
I think the validate-fixture-* need dash check too

Reply:
I have used a new feature from https://getrector.org/demo, using create PR. Code is unmodified from what has been generated by the tool.

Reply:
The filename is the problem